deploy/ansible/worker/README.md

```diff
@@ -0,0 +1,26 @@
+#### single command line worker deployment
+
```

Comment:
I'd like to see this move into a formalized guide in the official documentation.